content/pt-br/docs/concepts/scheduling-eviction/_index.md

```diff
@@ -1,8 +1,30 @@
 ---
-title: "Escalonamento"
-weight: 90
+title: "Escalonamento, preempção e remoção"
+weight: 95
+content_type: concept
 description: >
-   No Kubernetes, agendamento refere-se a garantia de que os pods correspondam aos nós para que o kubelet possa executá-los.
-   Remoção é o processo de falha proativa de um ou mais pods em nós com falta de recursos.
+  No Kubernetes, escalonamento refere-se a certeza de que os Pods correspondam aos nós para que o kubelet possa executá-los. Preempção é o processo de finalizar Pods com menor prioridade, para que os Pods com maior prioridade possam ser escalonados nos nós. Remoção é o processo de finalização proativa de um ou mais Pods em nós com poucos recursos.
+no_list: true
 ---
 
+No Kubernetes, escalonamento refere-se a certeza de que {{<glossary_tooltip text="Pods" term_id="pod">}}
+correspondam aos {{<glossary_tooltip text="nós" term_id="node">}} para que o
+{{<glossary_tooltip text="Kubelet" term_id="kubelet">}} possa executá-los. Preempção é o processo de finalizar Pods com menor {{<glossary_tooltip text="prioridade" term_id="pod-priority">}}
+para que os Pods com maior prioridade possam ser escalonados nos nós. Remoção é o processo de finalização proativa de um ou mais Pods em nós com poucos recursos.
+
+## Escalonamento
+
+* [Escalonador do Kubernetes](/pt-br/docs/concepts/scheduling-eviction/kube-scheduler/)
+* [Atribuindo Pods à Nós](/docs/concepts/scheduling-eviction/assign-pod-node/)
+* [Sobrecarga de Pod](/pt-br/docs/concepts/scheduling-eviction/pod-overhead/)
+* [Restrições de Propagação da Topologia do Pod](/docs/concepts/scheduling-eviction/topology-spread-constraints/)
+* [Taints e Tolerâncias](/pt-br/docs/concepts/scheduling-eviction/taint-and-toleration/)
+* [Framework do Escalonador](/docs/concepts/scheduling-eviction/scheduling-framework)
+* [Refinando a Performance do Escalonador](/docs/concepts/scheduling-eviction/scheduler-perf-tuning/)
+* [Empacotamento de Recursos para Recursos Estendidos](/docs/concepts/scheduling-eviction/resource-bin-packing/)
+
+## Interrupção do Pod
+
```

Comment:
There is a line (glossary_definition) here in the [original file](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/scheduling-eviction/_index.md).

Reply:
Done!